pom.xml

```diff
@@ -2337,7 +2337,7 @@
       </build>
     </profile>
     <profile>
-      <id>stable</id>
+      <id>stable-tests</id>
```

Comment:
Is this profile being used as of today ?

Reply:
I don't think so.  Probably can be removed in some cleanup task.